pkgs/tools/graphics/pfstools/default.nix

```diff
@@ -1,24 +1,28 @@
-{stdenv, fetchurl, libtiff, openexr, imagemagick, libjpeg, qt4, mesa,
-freeglut, bzip2, libX11, libpng, expat, pkgconfig }:
+{ stdenv, fetchurl, cmake, pkgconfig, openexr, ilmbase, zlib, imagemagick, mesa, freeglut, fftwFloat, fftw, gsl, libexif, perl, opencv, qt4 }:
 
 stdenv.mkDerivation rec {
-  name = "pfstools-1.8.5";
+  name = "pfstools";
+  version = "2.0.5";
 
   src = fetchurl {
-    url = "mirror://sourceforge/pfstools/${name}.tar.gz";
-    sha256 = "01kk2r8afrb3vrhm8abfjdhhan97lzpapc4n8w1mpdp3kv9miy9c";
+    url = "http://downloads.sourceforge.net/project/pfstools/pfstools/${version}/${name}-${version}.tgz";
+    sha256 = "1fyc2c7jzr7k797c2dqyyvapzc3szxwcp48r382yxz2yq558xgd9";
   };
 
-  configureFlags = "--with-moc=${qt4}/bin/moc";
+  cmakeFlags = ''
+    -DWITH_MATLAB=false 
+  '';
 
-  buildInputs = [ libtiff openexr imagemagick libjpeg qt4 mesa freeglut
-    bzip2 libX11 libpng expat ];
+  buildInputs = [ openexr zlib imagemagick mesa freeglut fftwFloat fftw gsl libexif perl opencv qt4 ];
 
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ cmake pkgconfig ];
 
-  meta = {
-    homepage = http://pfstools.sourceforge.net/;
+  patches = [ ./threads.patch ./pfstools.patch ];
```

Comment:
@juliendehos would it be possible to fetch this patches (using fetchurl) from some url? this way next person will know where to look for newer patches.